src/main/resources/db/migration/V3_17__update_application_and_visit_contact_number_to_null_where_unknown.sql

```diff
@@ -0,0 +1,2 @@
+update application_contact set contact_phone = NULL where upper(contact_phone) = 'UNKNOWN';
+update visit_contact set contact_phone = NULL where upper(contact_phone) = 'UNKNOWN';
```

Comment:
Why uppper should always be upper case?

Reply:
will leave it. just an additional check.